src/introduction/why_mit.md

```diff
@@ -3,21 +3,28 @@ Why MIT?
 
 As licensing is rather controversial, this is a frequently asked question.
 
-The GPL is good for forcing people who make changes to the source to contribute back. This would be okay if you were developing a program, but never ideal for libraries or operating systems, because the GPL forces any code that even remotely uses or links to the GPL'd source, to become GPL'd.
+The GPL is good for forcing people who make changes to the source to contribute back. GPL stipulates that source code that is modified, compiled, and then distributed must be published under the GPL license. This prevents a company like Google, for example, from keeping most modifications to the Linux kernel for Android private.
 
-Since operating systems are such an integrated part of our lives, we want as little restriction as possible. Restrictive licenses are quite a big disadvantage for OS projects. The MIT license opens up a lot of possibilities, which are simply not plausible with, say, the GPL.
+Since operating systems are such an integrated part of our lives, we want as little restriction as possible.
 
-The GPL is upstream-centric, the MIT license is downstream-centric. We happen to prioritize downstream more than upstream, since downstream is what really matters: the userbase, the community, the availability.
```

Comment:
It seems like an opinion, and one that was triggering a lot of people who say things like:

GPL protects the downstream by making all distributed contributions public

Which is, of course, just another opinion that we disagree with